interface/ceed-elemrestriction.c

```diff
@@ -866,15 +866,16 @@ int CeedElemRestrictionGetMultiplicity(CeedElemRestriction rstr,
   int ierr;
   CeedVector e_vec;
 
-  // Create and set e_vec
+  // Create e_vec to hold intermediate computation in E^T (E 1)
   ierr = CeedElemRestrictionCreateVector(rstr, NULL, &e_vec); CeedChk(ierr);
-  ierr = CeedVectorSetValue(e_vec, 1.0); CeedChk(ierr);
-  ierr = CeedVectorSetValue(mult, 0.0); CeedChk(ierr);
+  ierr = CeedVectorSetValue(mult, 1.0); CeedChk(ierr);
```

Comment:
Minor quibble - the comment above now doesn't really apply to both lines here. You could adjust the comment or just move line 871 down to the next block of code?

Reply:
Thanks, How about now?